src/Bicep.Cli/Program.cs

```diff
@@ -44,7 +45,12 @@ public static int Main(string[] args)
 
         public int Run(string[] args)
         {
-            ServiceProvider serviceProvider = ConfigureServices();
+            var serviceProvider = ConfigureServices();
+
+            if (bool.TryParse(Environment.GetEnvironmentVariable("BICEP_TRACING_ENABLED"), out var enableTracing) && enableTracing)
+            {
+                Trace.Listeners.Add(new TextWriterTraceListener(this.invocationContext.OutputWriter));
```

Comment:
>OutputWriter [](http://example.com/codeflow?start=87&length=12)

What happens with the msbuild task when the tracing is enabled?

Reply:
![boom](https://media.giphy.com/media/EPPvrXLVm6Axy/giphy.gif)

Reply:
I suspect this part needs to be changed in the msbuild task:
![image](https://user-images.githubusercontent.com/22460039/128090945-68422ff1-d7bc-4ebc-9ea6-1742ba113b28.png)

Trace messages can be logged in msbuild as a "message" instead of a warning, error, or an informational thing. But you may need to add a way to distinguish between a trace message vs some other unhandled exception.

Reply:
Done, good point.